lib/src/test/java/tech/relaycorp/relaydroid/messaging/SendMessageTest.kt

```diff
@@ -0,0 +1,84 @@
+package tech.relaycorp.relaydroid.messaging
+
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.check
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.whenever
+import kotlinx.coroutines.test.TestCoroutineScope
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Assert.assertArrayEquals
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import tech.relaycorp.poweb.PoWebClient
+import tech.relaycorp.relaydroid.PublicThirdPartyEndpoint
+import tech.relaycorp.relaydroid.test.FirstPartyEndpointFactory
+import tech.relaycorp.relaydroid.test.MessageFactory
+import tech.relaycorp.relaynet.bindings.pdc.ServerConnectionException
+import tech.relaycorp.relaynet.messages.Parcel
+import java.time.Duration
+
+internal class SendMessageTest {
+
+    private val poWebClient = mock<PoWebClient>()
```

Comment:
This is not using the testing lib, which we agreed to use: https://github.com/relaycorp/relaynet-jvm-testing#mock-pdc-client